examples/howto/plot_connectivity.py

```diff
@@ -115,16 +120,16 @@ def get_network(probability=1.0):
 
 ###############################################################################
 # We can plot the sparse connectivity pattern between cell populations.
-conn_idx = len(net_sparse.connectivity) - 1
-plot_connectivity_matrix(net_sparse, conn_idx)
-
-conn_idx = len(net_sparse.connectivity) - 2
+src_gids, target_gids = 'L2_basket', 'L2_basket'
+loc, receptor = 'soma', 'gabaa'
+conn_indices = net_erp.pick_connection(src_gids, target_gids, loc, receptor)
+conn_idx = conn_indices[0]
```

Comment:
much better! now only thing is that you shouldn't have to do this:

```suggestion
```

so `plot_connectivity_matrix` accepts a list. Can be a separate PR though. Are there other functions which benefit from `pick_connection` ?

Reply:
I agree that'd be a bit nicer, but what do you think the alternate behavior should be with a list? Returning multiple plots or subplots?

Reply:
umm ... either is fine. Maybe subplots is better so user is not overwhelmed by plot windows. Each row is one connectivity matrix if multiple subplots. And you can even implement a [scrollbar](https://matplotlib.org/stable/gallery/widgets/slider_demo.html#sphx-glr-gallery-widgets-slider-demo-py) if there are too many subplots